generate/generate.go

```diff
@@ -560,7 +560,7 @@ func (as *allServices) GeneralCode() ([]byte, error) {
 	pn("// Execute the request against a CS API. Will return the raw JSON data returned by the API and nil if")
 	pn("// no error occurred. If the API returns an error the result will be nil and the HTTP error code and CS")
 	pn("// error details. If a processing (code) error occurs the result will be nil and the generated error")
-	pn("func (cs *CloudStackClient) newRequest(api string, params url.Values) (json.RawMessage, error) {")
+	pn("func (cs *CloudStackClient) newPostRequest(api string, params url.Values) (json.RawMessage, error) {")
```

Comment:
@kevin-lii should all API calls be POST by default? Or, we should compare if it's a list/get API ?

cc @Pearl1594 @weizhouapache @sureshanaparti @vishesh92

Reply:
Sorry I think I accidentally replaced that. I will fix that.